### PR TITLE
Feat: Progress docs

### PR DESCRIPTION
**Pull request by Hugos68**
This PR documents the Progress component for both Svelte and React.

### CODE REVIEW

**Comment by endigo9740** [comment]
@Hugos68 I've disabled the Vercel setting that requires authentication to generate a preview deployment. Will you push a quick change to test this out please? Just change some text, etc.

File: sites/next.skeleton.dev/src/examples/components/progress/Example.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { Progress } from '@skeletonlabs/skeleton-react';
+
+export default () => {
+	return <Progress value={50} ariaLabelledby="A halfway filled progress bar" classes="w-full" />;
```
Comment (endigo9740) [inline sites/next.skeleton.dev/src/examples/components/progress/Example.tsx:4]: See the previous comment regarding `labelledBy`

File: sites/next.skeleton.dev/src/examples/components/progress/ExampleIndeterminate.svelte [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<script>
+	import { Progress } from '@skeletonlabs/skeleton-svelte';
+</script>
+
+<Progress value={undefined} ariaLabelledby="A halfway filled progress bar" classes="w-full"  />
```
Comment (endigo9740) [inline sites/next.skeleton.dev/src/examples/components/progress/ExampleIndeterminate.svelte:5]: See the previous comment regarding `labelledBy`

File: sites/next.skeleton.dev/src/examples/components/progress/ExampleIndeterminate.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { Progress } from '@skeletonlabs/skeleton-react';
+
+export default () => {
+	return <Progress value={undefined} ariaLabelledby="A halfway filled progress bar" classes="w-full" />;
```
Comment (endigo9740) [inline sites/next.skeleton.dev/src/examples/components/progress/ExampleIndeterminate.tsx:4]: See the previous comment regarding `labelledBy`

**Review by endigo9740 — CHANGES_REQUESTED** [review]
@Hugos68 I've tagged a few issues in the code, but here's a few additional suggestions:

- [ ] I'd suggest we make the Progress default color the neutral tone to match the native element. Though the track needs a bit of adjustment. Let me know if I can jump in and help with the cosmetics here.
- [ ] Let's add an example showing 2-3 size variations
- [ ] Let's add an example showing primary/secondary/tertiary variations
- [ ] We need documentation around using custom animations.

File: sites/next.skeleton.dev/src/examples/components/progress/Example.svelte [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<script>
+	import { Progress } from '@skeletonlabs/skeleton-svelte';
+</script>
+
+<Progress value={50} ariaLabelledby="A halfway filled progress bar" classes="w-full" />
```
[the remark this answers is not in the document]
Comment (Hugos68) [reply sites/next.skeleton.dev/src/examples/components/progress/Example.svelte:5]: Oh right, it's labelledby, not label, thanks, ill exclude it from the examples

File: sites/next.skeleton.dev/src/examples/components/progress/Example.tsx [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import { Progress } from '@skeletonlabs/skeleton-react';
+
+export default () => {
```
Comment (endigo9740) [inline sites/next.skeleton.dev/src/examples/components/progress/Example.tsx:3]: The React examples will need to be updated to follow this page-like structure we use in all existing component examples:

https://github.com/skeletonlabs/skeleton/blob/next/sites/next.skeleton.dev/src/examples/components/avatars/Example.tsx#L4

You can then import as follows:

```ts
import { Page as Example } from '@examples/components/avatars/Example.tsx';
import ExampleRaw from '@examples/components/avatars/Example.tsx?raw';
```

**Review by endigo9740 — CHANGES_REQUESTED** [review]

**Review by Mahmoud-zino — APPROVED** [review]
nicely done :ok_hand:

**Pull request merged**